src/Stratis.Bitcoin.Features.Consensus/ProvenBlockHeaders/IProvenBlockHeaderStore.cs

```diff
@@ -9,6 +10,39 @@ namespace Stratis.Bitcoin.Features.Consensus.ProvenBlockHeaders
     /// </summary>
     public interface IProvenBlockHeaderStore : IProvenBlockHeaderProvider
     {
+        /// <summary>
+        /// Initializes the <see cref="ProvenBlockHeaderStore"/>.
```

Comment:
reference IProvenBlockHeaderStore, not implementation

Reply:
Ok.